pages/api/utils/rollBar.ts

```diff
@@ -10,6 +10,12 @@ const rollbar = new Rollbar({
   captureUncaught: true,
   captureUnhandledRejections: true,
   enabled: isRollBarEnabled,
+  checkIgnore: (_, args) => {
```

Comment:
Is this necessary? This is the server-side Rollbar config, right? I'm pretty sure that hydration errors only happen on the client and wouldn't show up here.

Reply:
I will check again, but I added it here as I was getting the error `Expected server HTML` and wanted to ensure we don't pass this to Rollbar. We might be able to remove.

Reply:
I don't see any occurrences of that error in Rollbar